README.md

```diff
@@ -5,6 +5,10 @@ Budget Manager is a Ruby on Rails capstone project about building a mobile web a
 * Ruby
 * Rails
 * Postgresql
+
+[zoom Video](https://drive.google.com/file/d/1UyQRIaSCpa0_XA_WQLkAGXXtFAaCz2PR/view?usp=sharing)
+
+[live Demo](https://true-lumberjack-62637.herokuapp.com/)
```

Comment:
- [ ] kindly note that one of the requirements says that you should deploy your app and add a valid link for in in the `README`. Kindly provide a valid link here because the current one is broken

![image](https://user-images.githubusercontent.com/65068771/176977800-653a9d25-c051-4fe3-bbcc-f1cd0f8561c3.png)

- [ ] Make sure also that your README contains an "Acknowledgements" section or similar that gives appropriate credit to Gregoire Vella on Behance the author of the original design, as required by the Creative Commons license of the design.

Reply:
I have updated and made the requested changes. Thanks @andersonlebon